bot/discord.go

```diff
@@ -115,26 +116,26 @@ func dMessageCreate(s *discord.Session, m *discord.MessageCreate) {
 			replace := fmt.Sprintf("@%s", r.Name)
 			message = strings.Replace(message, find, replace, -1)
 		}
-
+        
 		// Multiline
 		lines := strings.Split(message, "\n")
 		if len(lines) > 3 {
 			url := uploadToPtpb(message)
 
 			for _, line := range lines[:2] {
-				incomingDiscord(m.Author.Username, channel, line)
+				incomingDiscord(authorName, channel, line)
 			}
-			incomingDiscord("[SYSTEM]", channel, fmt.Sprintf("full message from %s: %s", iAddAntiPing(m.Author.Username), url))
+			incomingDiscord("[SYSTEM]", channel, fmt.Sprintf("full message from %s: %s", iAddAntiPing(authorName), url))
 		} else {
 			for _, line := range lines {
-				incomingDiscord(m.Author.Username, channel, line)
+				incomingDiscord(authorName, channel, line)
 			}
 		}
 
-		//incomingDiscord(m.Author.Username, channel, message)
+		//incomingDiscord(authorName, channel, message)
```

Comment:
I saw no reason to leave the comment untouched. Either I update it or I remove it.